### PR TITLE
Improve code block formatting in Getting Started guide - Local Install

### PR DESCRIPTION
**Pull request by mhmohona**
I have noticed the inconsistency of the code block formatting which confused me as a new user while going through the [local installation](https://vitess.io/docs/20.0/get-started/local/) guide.  

Screenshot from 2024-08-01 09-45-20

Related Issue: #1738

### CODE REVIEW

**Review by mattlord — APPROVED** [review]
Thanks @mhmohona ! ❤️

**Review by mattlord — CHANGES_REQUESTED** [review]
Thanks, @mhmohona ! I confirmed that it all looks good here: https://deploy-preview-1803--vitess.netlify.app/docs/20.0/get-started/local/

The only thing we need to do is make the same changes in the 21.0 docs, otherwise the changes in v20 will eventually be lost when that version is EOL'd.

**Comment by mhmohona** [comment]
Thank you for pointing that @mattlord! Updating it right away. 

Another thing I think we can improve is, currently both code block and output block is in dark color. As new user, the output block really helped me, but I think if the output block, like the one in [Start a Single Keyspace Cluster](https://vitess.io/docs/20.0/get-started/local/#start-a-single-keyspace-cluster) section was in different color, it would be more understandable for user to differentiate. What do you think @mattlord?

**Comment by mattlord** [comment]
> Another thing I think we can improve is, currently both code block and output block is in dark color. As new user, the output block really helped me, but I think if the output block, like the one in [Start a Single Keyspace Cluster](https://vitess.io/docs/20.0/get-started/local/#start-a-single-keyspace-cluster) section was in different color, it would be more understandable for user to differentiate. What do you think @mattlord?

Yeah, I'm certainly open to that. I think the only consideration is that the contrast is relatively high for people with visual impairment.

File: content/en/docs/21.0/get-started/local.md [hunk 1]
```diff
@@ -74,22 +76,23 @@ sudo aa-status | grep mysqld
...
+
 ```sh
 # CentOS
 sudo setenforce 0
 ```
 
 ## Install Vitess
 
-Download the [latest binary release](https://github.com/vitessio/vitess/releases) for Vitess on Linux. For example:
+Download the [latest binary release](https://github.com/vitessio/vitess/releases) for Vitess on Linux. For example with Vitess 19:
```
Comment (mattlord) [inline content/en/docs/21.0/get-started/local.md]: This should say Vitess 20.

**Review by mattlord — APPROVED** [review]

**Pull request merged**